packages/components/src/locales/en-us/messages.json

```diff
@@ -686,5 +686,20 @@
     "message": "Show Language Selector",
     "description": "Tooltip for Show Languges Titles Block",
     "component": "Grammar"
+  },
+  "LOOKUP_USE_BETA_CODES_CHECK": {
+    "message": "update beta codes",
```

Comment:
please change the text to  "Enable input in betacode"

Reply:
fixed
![image](https://user-images.githubusercontent.com/12377640/78735011-09ccfc80-798d-11ea-956d-bbc743ab9d40.png)